### PR TITLE
fix(DDI-1276): Form stepper hover updates

### PR DESCRIPTION
**Pull request by amar-ABgov**

### CODE REVIEW

File: libs/web-components/src/components/form-step/FormStep.svelte [hunk 1]
```diff
@@ -153,7 +154,7 @@
   aria-current={_isCurrent ? "step" : "false"}
   aria-label={`${arialabel} ${text} ${status || ""}`}
 >
-  <input id={text} bind:this={_checkbox} type="checkbox" checked={_isCurrent} disabled={!_isEnabled} />
+  <input id={text} bind:this={_checkbox} type="checkbox" checked={_isCurrent} aria-disabled={_isEnabled}/>
```
Comment (chrisolsen) [inline libs/web-components/src/components/form-step/FormStep.svelte:157]: Should this be?
```
aria-disabled={!_isEnabled}
```

**Review by ArakTaiRoth — DISMISSED** [review]

**Review by chrisolsen — APPROVED** [review]

**Review by ArakTaiRoth — APPROVED** [review]

**Pull request merged**